x/evm/keeper/msg_server.go

```diff
@@ -37,6 +38,10 @@ type msgServer struct {
 // TODO: make this a param when we can easily switch between different kinds of keys and different settings
 var keyRole = tss.SecondaryKey
 
+const (
+	voteCostPerMaintainer = uint64(120000)
```

Comment:
I'd think this should be part of the vote module and happens when poll is initialized. This makes me think that we should bump gas cost for the initialization of any kind of interactive process.